Tamil/kambaramayanam/into.txt

```diff
@@ -0,0 +1,4 @@
+.	??? ?????? ???? ???? ???? ????? ???
```

Comment:
It also looks like the encoding got messed up somehow. All I see is question marks.